src/servers/Server_Zone/Network/Handlers/GMCommandHandlers.cpp

```diff
@@ -184,30 +182,27 @@ void Core::Network::GameConnection::gm1Handler( const Packets::GamePacket& inPac
    {
       targetPlayer->getCurrentZone()->setWeatherOverride( param1 );
       player.sendNotice( "Weather in Zone \"" + targetPlayer->getCurrentZone()->getName() + "\" of " +
-         targetPlayer->getName() + " set in range." );
+                         targetPlayer->getName() + " set in range." );
       break;
    }
    case GmCommand::Call:
    {
-      if ( targetPlayer->getZoneId() != player.getZoneId() )
+      if( targetPlayer->getZoneId() != player.getZoneId() )
          targetPlayer->setZone( player.getZoneId() );
 
-      targetPlayer->changePosition( player.getPos().x, player.getPos().y, player.getPos().z,
-         player.getRotation() );
+      targetPlayer->changePosition( player.getPos().x, player.getPos().y, player.getPos().z, player.getRotation() );
       player.sendNotice( "Calling " + targetPlayer->getName() );
       break;
    }
    case GmCommand::Inspect:
    {
-      player.sendNotice( "Name: " + targetPlayer->getName() +
-         "\nGil: " + std::to_string( targetPlayer->getCurrency( 1 ) ) +
-         "\nZone: " + targetPlayer->getCurrentZone()->getName() +
-         "(" + std::to_string( targetPlayer->getZoneId() ) + ")" +
-         "\nClass: " + std::to_string( static_cast< uint8_t >( targetPlayer->getClass() ) ) +
-         "\nLevel: " + std::to_string( targetPlayer->getLevel() ) +
-         "\nExp: " + std::to_string( targetPlayer->getExp() ) +
-         "\nSearchMessage: " + targetPlayer->getSearchMessage() +
-         "\nPlayTime: " + std::to_string( targetPlayer->getPlayTime() ) );
+      player.sendNotice(
+          "Name: " + targetPlayer->getName() + "\nGil: " + std::to_string( targetPlayer->getCurrency( 1 ) ) +
+          "\nZone: " + targetPlayer->getCurrentZone()->getName() + "(" + std::to_string( targetPlayer->getZoneId() ) +
+          ")" + "\nClass: " + std::to_string( static_cast< uint8_t >( targetPlayer->getClass() ) ) + "\nLevel: " +
+          std::to_string( targetPlayer->getLevel() ) + "\nExp: " + std::to_string( targetPlayer->getExp() ) +
+          "\nSearchMessage: " + targetPlayer->getSearchMessage() +
+          "\nPlayTime: " + std::to_string( targetPlayer->getPlayTime() ) );
```

Comment:
this is basically worse than what we had but there's probably nothing that can be done without rewriting this or disabling formatting for this one part right?

Reply:
Yea, it wont be able to properly format that automatically.